src/browser/BrowserService.cpp

```diff
@@ -284,6 +283,30 @@ QJsonObject BrowserService::createNewGroup(const QString& groupName)
     return result;
 }
 
+QString BrowserService::getCurrentTotp(const QString& uuid)
+{
+    QList<QSharedPointer<Database>> databases;
+    if (browserSettings()->searchInAllDatabases()) {
+        for (auto dbWidget : getMainWindow()->getOpenDatabases()) {
+            auto db = dbWidget->database();
+            if (db) {
+                databases << db;
+            }
+        }
```

Comment:
Future improvement... we need to make it dead simple to get the current valid database objects.

Reply:
I'll put that to my TODO list.